tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2091,6 +2106,32 @@ public function createPublicLinkToEntityInsideOfSpaceRequest(
 				$body
 			)
 		);
+
+		$this->setLastShareData();
+	}
+
+	/**
+	 * @When /^user "([^"]*)" has created a public link share inside of space "([^"]*)" with settings:$/
+	 *
+	 * @param  string $user
+	 * @param  string $spaceName
+	 * @param TableNode|null $table
```

Comment:
```suggestion
	 * @param TableNode $table
```
this should never be `null`, same in `createPublicLinkToEntityInsideOfSpaceRequest` as it calls `$rows = $table->getRowsHash();`